etna/transforms/missing_values/resample.py

```diff
@@ -61,8 +61,8 @@ def _get_folds(self, df: pd.DataFrame) -> List[int]:
         n_periods = len(df) // n_folds_per_gap + 2
 
         in_column_start_index = in_column_index[0]
-        left_tie_len = len(df[:in_column_start_index]) - 1
-        right_tie_len = len(df[in_column_start_index:])
+        left_tie_len = len(df.loc[:in_column_start_index]) - 1
```

Comment:
Do we have tests for this?)

Reply:
I changed fixture for resampling in inference tests to check this.